Sources/NIOSSL/IdentityVerification.swift

```diff
@@ -14,6 +14,10 @@
 
 import NIOCore
 
+#if os(Linux) || os(FreeBSD) || os(Android)
+import CNIOLinux
+#endif
```

Comment:
You should be able to unconditionally import this -- it if-defs itself so that it's empty on other platforms.

Reply:
Thanks, I forgot I did that here